isso: build failing: Error: Cannot find module 'character-parser'

### Describe the bug
The build for `isso` fails. See log:

<details>
<summary><code>nix-build -A isso</code> output</summary>

```
$ nix-build -A isso
this derivation will be built:
  /nix/store/gm5c87fv0bd7c4yxqf9p5849s2zabqkr-isso-0.12.5.drv
these 10 paths will be fetched (1.43 MiB download, 6.96 MiB unpacked):
  /nix/store/48c9r6zqbb06qqhks06iqhfz0hy418fb-python3.9-itsdangerous-2.0.1
  /nix/store/55gl146996h1j1gj5mydv3m7imq932kn-python3.9-asgiref-3.4.1
  /nix/store/979a8jp4caickc5rybc728mpmwy2sq8g-python3.9-nose-1.3.7
  /nix/store/dixgra62cj6ysssdxyypgpibks7gjh5y-python3.9-werkzeug-2.0.2
  /nix/store/f2finmmpbfnplh0qbr3rs3xidm42hb7n-source
  /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest
  /nix/store/pfmlxqr87r6bv7cz3zdlhgbsyiy7x6gi-python3.9-coverage-5.5
  /nix/store/r389ldsjp40pwkq9q99fa7lnrrk51ljd-python3.9-misaka-2.1.1
  /nix/store/vf78d6b7s1wk59a8xlgv0dlq4izixjyv-python3.9-Flask-2.0.2
  /nix/store/z6aqpbwj7pkm0d7hrgkf9h50cqpb6la3-python3.9-Flask-Caching-1.10.1
copying path '/nix/store/f2finmmpbfnplh0qbr3rs3xidm42hb7n-source' from 'https://cache.nixos.org'...
copying path '/nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest' from 'https://cache.nixos.org'...
copying path '/nix/store/48c9r6zqbb06qqhks06iqhfz0hy418fb-python3.9-itsdangerous-2.0.1' from 'https://cache.nixos.org'...
copying path '/nix/store/r389ldsjp40pwkq9q99fa7lnrrk51ljd-python3.9-misaka-2.1.1' from 'https://cache.nixos.org'...
copying path '/nix/store/dixgra62cj6ysssdxyypgpibks7gjh5y-python3.9-werkzeug-2.0.2' from 'https://cache.nixos.org'...
copying path '/nix/store/pfmlxqr87r6bv7cz3zdlhgbsyiy7x6gi-python3.9-coverage-5.5' from 'https://cache.nixos.org'...
copying path '/nix/store/55gl146996h1j1gj5mydv3m7imq932kn-python3.9-asgiref-3.4.1' from 'https://cache.nixos.org'...
copying path '/nix/store/979a8jp4caickc5rybc728mpmwy2sq8g-python3.9-nose-1.3.7' from 'https://cache.nixos.org'...
copying path '/nix/store/vf78d6b7s1wk59a8xlgv0dlq4izixjyv-python3.9-Flask-2.0.2' from 'https://cache.nixos.org'...
copying path '/nix/store/z6aqpbwj7pkm0d7hrgkf9h50cqpb6la3-python3.9-Flask-Caching-1.10.1' from 'https://cache.nixos.org'...
building '/nix/store/gm5c87fv0bd7c4yxqf9p5849s2zabqkr-isso-0.12.5.drv'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Using setuptoolsShellHook
Sourcing pip-install-hook
Using pipInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing setuptools-check-hook
unpacking sources
unpacking source archive /nix/store/f2finmmpbfnplh0qbr3rs3xidm42hb7n-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/uwsgi.ini
patching sources
configuring
no configure script, doing nothing
building
Executing setuptoolsBuildPhase
sh: line 1: git: command not found
npx --no-install r.js -o isso/js/build.embed.js out=isso/js/embed.min.js

Tracing dependencies for: ../../node_modules/almond/almond
Error: Cannot find module 'character-parser'
Require stack:
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/jade/lib/lexer.js
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/jade/lib/parser.js
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/jade/lib/index.js
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/requirejs/bin/r.js
In module tree:
    embed
      app/isso
        app/jade
          jade

Error: Error: Cannot find module 'character-parser'
Require stack:
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/jade/lib/lexer.js
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/jade/lib/parser.js
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/jade/lib/index.js
- /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/requirejs/bin/r.js
In module tree:
    embed
      app/isso
        app/jade
          jade

    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)

make: *** [Makefile:46: isso/js/embed.min.js] Error 1
error: builder for '/nix/store/gm5c87fv0bd7c4yxqf9p5849s2zabqkr-isso-0.12.5.drv' failed with exit code 2;
       last 10 log lines:
       > - /nix/store/g6gss8pgi1k144sjmsi8qhk133if9yhc-node-dependencies-isso-latest/lib/node_modules/requirejs/bin/r.js
       > In module tree:
       >     embed
       >       app/isso
       >         app/jade
       >           jade
       >
       >     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
       >
       > make: *** [Makefile:46: isso/js/embed.min.js] Error 1
       For full logs, run 'nix log /nix/store/gm5c87fv0bd7c4yxqf9p5849s2zabqkr-isso-0.12.5.drv'.
```

</details>

### Steps To Reproduce

Run `nix-build -A isso` on latest `master`

### Expected behavior

It builds

### Notify maintainers

@fgaz and @asbachb since you added  the JS generation code

### Related PRs

https://github.com/NixOS/nixpkgs/pull/132974

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.87, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.6.0pre20211217_6e6e998`
 - channels(root): `"nixos"`
 - channels(sumner): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17268,7 +17268,9 @@ with pkgs;
     llvmPackages = llvmPackages_11;
   };
 
-  isso = callPackage ../servers/isso { };
+  isso = callPackage ../servers/isso {
+    nodejs = nodejs-14_x;
+  };
 
   itk4 = callPackage ../development/libraries/itk/4.x.nix {
     inherit (darwin.apple_sdk.frameworks) Cocoa;

